### ISSUE TITLE
crash on launch - bad json format

### ISSUE THREAD

**Issue by PapyGaragos** [SBBTracker/SBBTracker#26]
The app was working fine yesterday, and did not update today before the error occured. 
I manually installed the latest release and still have the issue
I cannot locate any `settings.json` file in the installation folder.

```
Traceback (most recent call last):
  File "application.py", line 131, in <module>
  File "application.py", line 126, in load_settings
  File "json\__init__.py", line 293, in load
  File "json\__init__.py", line 346, in loads
  File "json\decoder.py", line 337, in decode
  File "json\decoder.py", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by andrewrasch** [comment]
Hi, the settings.json file is actually at Documents/SBBTracker/settings.json. If you don't mind uploading it here so I can look at it, I would appreciate it.

**Comment by PapyGaragos** [comment]
It's an empty file, 0 Bytes.

**Comment by andrewrasch** [comment]
Yeah I'm unsure how it got into the empty state but good to know deleting it fixed it. I will add better handling of that in the next patch, thanks for reporting it.

**Comment by andrewrasch** [comment]
Fixed in v3.1.0
